tests/shared/DevelopmentTestDefs.js

```diff
@@ -59,5 +58,8 @@ gpii.tests.development.testDefs = [{
     }, {
         event: "{logoutRequest}.events.onComplete",
         listener: "gpii.tests.development.testLogoutResponse"
-    }]
+    }],
+    events: {
+        timeoutComplete: null
```

Comment:
This event is unclearly named and the reader will not be able to determine what the timeout relates to